doc/source/news.rst

```diff
@@ -5,6 +5,48 @@
 News
 ====
 
+.. _release-12-0-9:
+
+Release 12.0.9 - 2022-10-28
+---------------------------
+
+Improvements
+------------
+
+* [:doc:`reference/functions/escalate`] Added a document for the ``escalate()`` function.
+
+* [:doc:`reference/normalizers`] Added NormalizerHTML. (Experimental)
+
+  This is a normalizer for HTML.
+
+  Currently this supports only unescaping reference characters like ``&amp;`` or ``&#38;``.
+
+  Here is a sample query for NormalizerHTML.
+
+  .. code-block::
+  
+     normalize NormalizerHTML "Groonga &amp; Mroonga &#38; Rroonga"
+     [[0,1666854852.063215,0.0006151199340820312],{"normalized":"Groonga & Mroonga & Rroonga","types":[],"checks":[]}]
+  
+  In this sample, ``&amp;`` and ``&#38;`` are unescaped to ``&``.
+
+* [httpd] Updated bundled nginx to 1.23.2.
+
+Fixes
+-----
+
+* [:doc:`reference/commands/select`] Fixed a bug that Groonga could crash or return incorrect results when specifying :ref:`select-n-workers`.
+
+  This had occured when using :ref:`select-n-workers` with a value greater than ``1`` and ``drilldowns[{LABEL}].filter`` at the same time.
+
+  This was because referencing incorrect values (objects) when performing internal parallel processing.
+  So if the condition above was satisfied, Groonga sometimes crashed or returned incorrect results depending on the timing of the parallel processing.
+
+* Suppressed logging a lot of same messages when no memory available.
```

Comment:
I think this is not a bug.
Because it is not abnormaly that Groonga output a lot of same messages when no memory available.

Reply:
Moved to Improvement.